mlflow/types/chat.py

```diff
@@ -0,0 +1,130 @@
+from __future__ import annotations
+
+from typing import Literal, Optional, Union
+
+from pydantic import BaseModel as _BaseModel
+from pydantic import Field
+from typing_extensions import Annotated
```

Comment:
MLflow doesn't depend on `typing_extensions`. We should add it as a dependency if we use it.

Reply:
In which python version was `Annotated` introduced?